--- rlocate-0.5.6/src/Makefile.am.orig	2007-08-15 10:57:33.000000000 +0200
+++ rlocate-0.5.6/src/Makefile.am	2016-09-24 20:24:27.153185828 +0200
@@ -28,7 +28,7 @@
 rlocate_SOURCES = pidfile.h pidfile.c slocate.c slocate.h \
 		  rlocate.h rlocate.c cmds.c cmds.h conf.c conf.h utils.c \
 	   	  utils.h
-SUBDIRS = rlocate-daemon rlocate-module rlocate-scripts
+SUBDIRS = rlocate-daemon rlocate-scripts
 EXTRA_DIST = rlocate.cron rlocate-scripts install-cron.sh.in
 
 INSTALL = install -c
@@ -36,10 +36,6 @@
 
 install-exec-hook:
 	$(INSTALL) -d $(RLOCATE_DB_DIR)
-	chown $(root_uid):$(rlocate_group) $(DESTDIR)$(bindir)/$(rlocate)
-	chown $(root_uid):$(rlocate_group) $(RLOCATE_DB_DIR)
-	chmod 2755 $(DESTDIR)$(bindir)/$(rlocate)
-	chmod 0750 $(RLOCATE_DB_DIR)
 	-$(LN_S) -f $(rlocate) $(DESTDIR)$(bindir)/$(updatedb)
 	-$(LN_S) -f $(rlocate) $(DESTDIR)$(bindir)/$(locate)
 if CRON
--- rlocate-0.5.6/configure.ac.orig	2016-09-24 20:17:46.859857059 +0200
+++ rlocate-0.5.6/configure.ac	2016-09-24 20:29:03.723182677 +0200
@@ -30,13 +30,9 @@
 # rlocate
 #
 AC_ENABLE_UPDATES
-AC_PATH_KERNEL_SOURCE
-AC_PATH_MODULE
 AC_SANDBOXED
-AC_MAJOR_NUMBER
 AC_RLOCATE_GROUP
 AC_UPDATEDB_CONF
-AC_PATH_DEV
 AC_PATH_STATUS
 AC_CHECK_PERL
 
@@ -81,7 +77,6 @@
 AC_CONFIG_FILES([Makefile
           src/Makefile
           src/rlocate-daemon/Makefile
-          src/rlocate-module/Makefile
           src/rlocate-scripts/rlocate-checkpoint
           src/rlocate-scripts/Makefile
           doc/Makefile
--- rlocate-0.5.6/src/rlocate-scripts/Makefile.am.orig	2016-09-24 20:29:26.019849182 +0200
+++ rlocate-0.5.6/src/rlocate-scripts/Makefile.am	2016-09-24 20:31:18.149848053 +0200
@@ -17,10 +17,3 @@
 	cp $(srcdir)/rlocate-checkpoint.temp $(srcdir)/rlocate-checkpoint
 	rm $(srcdir)/rlocate-checkpoint.temp
 
-install-exec-hook:
-	-/usr/sbin/groupadd $(rlocate_group)
-	chown $(root_uid):$(rlocate_group) $(DESTDIR)$(bindir)/rlocate-checkpoint
-	chmod 2755 $(DESTDIR)$(bindir)/rlocate-checkpoint
-
-uninstall-hook:
-	-/usr/sbin/groupdel $(rlocate_group)
